{
  "uid" : "d894fc509ad205f2",
  "name" : "test_transfer_zero_tokens_without_approval",
  "fullName" : "tests.brownie.token.tests.test_transferFrom#test_transfer_zero_tokens_without_approval",
  "historyId" : "8dca4b5cc80f723e5acea8426c32b8c6",
  "time" : {
    "start" : 1639106457493,
    "stop" : 1639106457493,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
  "statusTrace" : "brownie_project = <Project 'TokenProject'>\naccounts = <brownie.network.account.Accounts object at 0x7fd053c4b580>\n\n    @pytest.fixture(scope=\"module\")\n    def token(brownie_project, accounts):\n>       return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n\ntests/brownie/token/tests/conftest.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py:600: in __call__\n    return tx[\"from\"].deploy(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:508: in deploy\n    receipt, exc = self._make_transaction(\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py:743: in _make_transaction\n    exc = VirtualMachineError(e)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = VirtualMachineError(ValueError({'code': -32002, 'message': 'Verifying nonce before send transaction: Error processing ...valid program argument', 'data': {'logs': ['/src/entrypoint.rs Invalid Ethereum transaction nonce: acc 11, trx 10']}}))\nexc = {'code': -32002, 'data': {'logs': ['/src/entrypoint.rs Invalid Ethereum transaction nonce: acc 11, trx 10']}, 'message': 'Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument'}\n\n    def __init__(self, exc: ValueError) -> None:\n        try:\n            exc = exc.args[0]\n        except Exception:\n            pass\n    \n        if isinstance(exc, dict) and \"message\" in exc:\n            if \"data\" not in exc:\n                raise ValueError(exc[\"message\"]) from None\n    \n            self.message: str = exc[\"message\"].rstrip(\".\")\n    \n            if isinstance(exc[\"data\"], str):\n                # handle parity exceptions - this logic probably is not perfect\n                if \"0x08c379a0\" in exc[\"data\"]:\n                    revert_type, err_msg = [i.strip() for i in exc[\"data\"].split(\"0x08c379a0\", 1)]\n                    err_msg = eth_abi.decode_abi([\"string\"], HexBytes(err_msg))\n                    err_msg = f\"{revert_type} '{err_msg}'\"\n                elif exc[\"data\"].endswith(\"0x\"):\n                    err_msg = exc[\"data\"][:-2].strip()\n                else:\n                    err_msg = exc[\"data\"]\n                raise ValueError(f\"{self.message}: {err_msg}\") from None\n    \n            try:\n                txid, data = next((k, v) for k, v in exc[\"data\"].items() if k.startswith(\"0x\"))\n            except StopIteration:\n>               raise ValueError(exc[\"message\"]) from None\nE               ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument\n\n../../../.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py:104: ValueError",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : true,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ {
    "name" : "token",
    "time" : {
      "start" : 1639106364330,
      "stop" : 1639106456797,
      "duration" : 92467
    },
    "status" : "broken",
    "statusMessage" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument\n",
    "statusTrace" : "  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/neon-compatibility/neon-compatibility/tests/brownie/token/tests/conftest.py\", line 49, in token\n    return brownie_project.Token.deploy(\"Test Token\", \"TST\", 18, 1e21, {'from': accounts[0]})\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/contract.py\", line 600, in __call__\n    return tx[\"from\"].deploy(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 508, in deploy\n    receipt, exc = self._make_transaction(\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/network/account.py\", line 743, in _make_transaction\n    exc = VirtualMachineError(e)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/brownie/exceptions.py\", line 104, in __init__\n    raise ValueError(exc[\"message\"]) from None\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Brownie"
  }, {
    "name" : "parentSuite",
    "value" : "tests.brownie.token.tests"
  }, {
    "name" : "suite",
    "value" : "test_transferFrom"
  }, {
    "name" : "host",
    "value" : "fv-az74-543"
  }, {
    "name" : "thread",
    "value" : "4947-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.brownie.token.tests.test_transferFrom"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 17,
        "skipped" : 0,
        "passed" : 42,
        "unknown" : 0,
        "total" : 59
      },
      "items" : [ {
        "uid" : "2591a58aed9de45a",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/193//#testresult/2591a58aed9de45a",
        "status" : "passed",
        "time" : {
          "start" : 1639096787319,
          "stop" : 1639096789966,
          "duration" : 2647
        }
      }, {
        "uid" : "d95efb2e7fbb12e5",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/192//#testresult/d95efb2e7fbb12e5",
        "status" : "passed",
        "time" : {
          "start" : 1639085346135,
          "stop" : 1639085348432,
          "duration" : 2297
        }
      }, {
        "uid" : "b65ebfd02b5ac39e",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/189//#testresult/b65ebfd02b5ac39e",
        "status" : "passed",
        "time" : {
          "start" : 1639070710507,
          "stop" : 1639070725274,
          "duration" : 14767
        }
      }, {
        "uid" : "9d4ca87ab15a0a67",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/190//#testresult/9d4ca87ab15a0a67",
        "status" : "broken",
        "statusDetails" : "ValueError: Gas estimation failed: 'Transaction simulation failed: Error processing Instruction 1: invalid account data for instruction'. This transaction will likely revert. If you wish to broadcast, you must set the gas limit manually.",
        "time" : {
          "start" : 1639074286404,
          "stop" : 1639074286404,
          "duration" : 0
        }
      }, {
        "uid" : "3dc858333989a0f5",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/187//#testresult/3dc858333989a0f5",
        "status" : "passed",
        "time" : {
          "start" : 1639053400010,
          "stop" : 1639053406427,
          "duration" : 6417
        }
      }, {
        "uid" : "b7a759552d093b0c",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/188//#testresult/b7a759552d093b0c",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1639056863886,
          "stop" : 1639056895614,
          "duration" : 31728
        }
      }, {
        "uid" : "412317502df6c0c2",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/186//#testresult/412317502df6c0c2",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1639045902090,
          "stop" : 1639045902090,
          "duration" : 0
        }
      }, {
        "uid" : "1de24a514e8df872",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/184//#testresult/1de24a514e8df872",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1639019990463,
          "stop" : 1639019990463,
          "duration" : 0
        }
      }, {
        "uid" : "cbb4f7e48b0e0d7a",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/183//#testresult/cbb4f7e48b0e0d7a",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1639010575329,
          "stop" : 1639010606916,
          "duration" : 31587
        }
      }, {
        "uid" : "7c19ef22f4862ec5",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/182//#testresult/7c19ef22f4862ec5",
        "status" : "passed",
        "time" : {
          "start" : 1638990714987,
          "stop" : 1638990717650,
          "duration" : 2663
        }
      }, {
        "uid" : "aa88ca9401496db2",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/177//#testresult/aa88ca9401496db2",
        "status" : "passed",
        "time" : {
          "start" : 1638970012646,
          "stop" : 1638970025871,
          "duration" : 13225
        }
      }, {
        "uid" : "5bdfc29d88d5e8de",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/181//#testresult/5bdfc29d88d5e8de",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1638981046488,
          "stop" : 1638981046488,
          "duration" : 0
        }
      }, {
        "uid" : "b1374b7c137801",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/180//#testresult/b1374b7c137801",
        "status" : "broken",
        "statusDetails" : "ValueError: Verifying nonce before send transaction: Error processing Instruction 1: invalid program argument",
        "time" : {
          "start" : 1638975844867,
          "stop" : 1638975844867,
          "duration" : 0
        }
      }, {
        "uid" : "3a21b31b19d9caa9",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/179//#testresult/3a21b31b19d9caa9",
        "status" : "passed",
        "time" : {
          "start" : 1638974694271,
          "stop" : 1638974696902,
          "duration" : 2631
        }
      }, {
        "uid" : "7be9260309bf0a43",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/178//#testresult/7be9260309bf0a43",
        "status" : "passed",
        "time" : {
          "start" : 1638973650049,
          "stop" : 1638973652663,
          "duration" : 2614
        }
      }, {
        "uid" : "f1247cb9bce95b40",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/176//#testresult/f1247cb9bce95b40",
        "status" : "passed",
        "time" : {
          "start" : 1638951338373,
          "stop" : 1638951344922,
          "duration" : 6549
        }
      }, {
        "uid" : "a35e7e0dc87c66e6",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/174//#testresult/a35e7e0dc87c66e6",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='proxy.teststand2.neontest.xyz', port=443): Max retries exceeded with url: /solana (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fadbd81f0d0>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1638932978727,
          "stop" : 1638932978727,
          "duration" : 0
        }
      }, {
        "uid" : "a48b4ca92338e35a",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/173//#testresult/a48b4ca92338e35a",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='proxy.teststand2.neontest.xyz', port=443): Max retries exceeded with url: /solana (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fb865a3b130>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1638923387663,
          "stop" : 1638923387663,
          "duration" : 0
        }
      }, {
        "uid" : "fb0024dcf54f1124",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/172//#testresult/fb0024dcf54f1124",
        "status" : "passed",
        "time" : {
          "start" : 1638913376833,
          "stop" : 1638913387274,
          "duration" : 10441
        }
      }, {
        "uid" : "4218d7052004330e",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/171//#testresult/4218d7052004330e",
        "status" : "broken",
        "statusDetails" : "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='proxy.testnet.neonlabs.org', port=443): Max retries exceeded with url: /solana (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f240840a520>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1638912580380,
          "stop" : 1638912580380,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "d894fc509ad205f2.json",
  "parameterValues" : [ ]
}